css/main.css

```diff
@@ -0,0 +1,803 @@
+* {
```

Comment:
- [ ] CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))

* No small break point as it is mobile first and the base styles are for the mobile screen. ([Screenshot on mobile first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
* medium: min-width around 768px
* large: min-width 1024px

So please fix your breakpoints considering the mobile-first approach.